boards/Board.mk

```diff
@@ -30,12 +30,12 @@ $(RCOBJS): $(ETCDIR)$(DELIM)%: %
 	$(Q) mkdir -p $(dir $@)
 	$(call PREPROCESS, $<, $@)
 
-$(ETCSRC): $(RCRAWS) $(RCOBJS)
+$(ETCSRC): $(addprefix $(BOARD_DIR)$(DELIM)src$(DELIM),$(RCRAWS)) $(RCOBJS)
 	$(foreach raw, $(RCRAWS), \
 	  $(shell rm -rf $(ETCDIR)$(DELIM)$(raw)) \
 	  $(shell mkdir -p $(dir $(ETCDIR)$(DELIM)$(raw))) \
-	  $(shell cp -rfp $(raw) $(ETCDIR)$(DELIM)$(raw)))
-	$(Q) genromfs -f romfs.img -d $(ETCDIR)$(DELIM)$(CONFIG_NSH_ROMFSMOUNTPT) -V "$(basename $<)"
+	  $(shell cp -rfp $(BOARD_DIR)$(DELIM)src$(DELIM)$(raw) $(ETCDIR)$(DELIM)$(raw)))
+	$(Q) genromfs -f romfs.img -d $(ETCDIR)$(DELIM)$(CONFIG_NSH_ROMFSMOUNTPT) -V "NSHInitVol"
```

Comment:
@pkarashchenko I think someone added CONFIG_NSH_ROMFSMOUNTPT but we already have CONFIG_NSH_ROMFSETC that seams to fit the same purpose

Reply:
> I think someone added CONFIG_NSH_ROMFSMOUNTPT but we already have CONFIG_NSH_ROMFSETC that seams to fit the same purpose

`CONFIG_NSH_ROMFSETC` is a `bool` option to `Mount a ROMFS filesystem at` `CONFIG_NSH_ROMFSMOUNTPT` path (that is a string defaulted to `/etc`)

Reply:
Sorry, during the git grep I messed the comment after the symbol in defconfig point to "/etc" and assumed it was equivalent.